src/api/chassis/controller/chassisControllerPid.cpp

```diff
@@ -126,7 +127,7 @@ void ChassisControllerPID::moveDistanceAsync(const QLength itarget) {
   turnPid->flipDisable(true);
   mode = distance;
 
-  const double newTarget = itarget.convert(meter) * scales.straight * gearsetRatioPair.ratio;
+  const double newTarget = itarget.convert(meter) * scales.straight;
```

Comment:
Is this change (and the one below) backwards compatible for people that don't call `withDimensions` with a `GearsetRatioPair`?

Reply:
Yes, if one does `.withDimensions(AbstractMotor::gearset::green, {{4.0_in,10_in}, imev5GreenTPR})` it will create with a gearsetRatioPair with a 1.0 gear ratio defaulted.

Reply:
I am not sure where the external ratio is accounted for, then. It seems like it is always discarded.

Reply:
@Octogonapus - you are correct. Latest commit should fix this. The chassisControllerPID now works physically with both external tracking wheels and configured to use the drive train as the odom source.

I've phyically tested this with both configurations